Composer/packages/client/src/pages/botProject/create-publish-profile/PublishProfileDialog.tsx

```diff
@@ -31,13 +31,30 @@ type PublishProfileDialogProps = {
   setPublishTargets: (targets: PublishTarget[], projectId: string) => Promise<void>;
 };
 
+type ExtensionState = AzureExtensionState; // can expand to pva like | PVAExtensionState
+
+type AzureExtensionState = {
+  subscriptionId: string;
+  resourceGroup: string;
+  hostName: string;
+  location: string;
+  luisLocation: string;
+  enableResources: any;
+  requireResources: any;
```

Comment:
nit: rename to the following:
requiredResources
enabledResources

use types instead of any.